tests/playbooks/tests_802_1x.yml

```diff
@@ -89,6 +89,50 @@
                   domain_suffix_match: example.com
         - name: "TEST: I can ping the EAP server"
           shell: ping -c1 203.0.113.1
+        - import_role:
+            name: linux-system-roles.network
+          vars:
+            network_connections:
+              - name: "{{ interface }}"
+                persistent_state: absent
+                state: absent
+        - name: >-
+            TEST: 802.1x profile with unencrypted private key and ca_path
```

Comment:
The only reason this test passes, is because NetworkManager will not do any certificate validation by default